ajax/libs/attrchange/package.json

```diff
@@ -11,6 +11,12 @@
     "onchange",
     "onpropertychange"
   ],
+  "keywords": [
```

Comment:
@x09326 I think you can reference the upstream.
Line 6-12 of https://github.com/meetselva/attrchange/blob/master/attrchange.jquery.json